CHANGELOG/CHANGELOG-1.6.md

```diff
@@ -1,5 +1,20 @@
 # Changelog 1.6
 
+## 1.6.1 - 2022-05-11
+
+### Added
+
+- Add [documentation](https://github.com/antrea-io/antrea/blob/main/docs/security.md#protecting-your-cluster-against-privilege-escalations) for the Antrea Agent RBAC permissions and how to restrict them using Gatekeeper/OPA. ([#3694](https://github.com/antrea-io/antrea/pull/3694),[@antoninbas])
+
+### Fixed
+
+- Clean up stale routes installed by AntreaProxy when ProxyAll is disabled. ([#3465](https://github.com/antrea-io/antrea/pull/3465), [@hongliangl])
+- Fix Services with port name fail to realize when trying to export/import them via Antrea Multi-cluster feature. ([#3561](https://github.com/antrea-io/antrea/pull/3561), [@luolanzone])
+- Fix handling of the "reject" packets generated by the Antrea Agent to avoid infinite looping. ([#3569](https://github.com/antrea-io/antrea/pull/3569), [@GraysonWu])
+- Fix DNS resolution error of antrea-agent on AKS by using `ClusterFirst` dnsPolicy. ([#3701](https://github.com/antrea-io/antrea/pull/3701), [@tnqn])
```

Comment:
antrea-agent -> Antrea Agent, to be consistent with other descriptions.

Reply:
updated.